organization/src/test/scala/com/improving/app/organization/utils/LoanedActorSystem.scala

```diff
@@ -0,0 +1,28 @@
+package com.improving.app.organization.utils
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import com.typesafe.config.ConfigFactory
+
+import java.util.concurrent.atomic.AtomicInteger
+import scala.jdk.CollectionConverters._
+
+trait LoanedActorSystem {
+  def configOverrides: Map[String, Any] = Map.empty
+
+  private val n = new AtomicInteger
```

Comment:
Is this even needed ? It doesn't seem referenced in this trait.

Reply:
It is used in the tests using trait CassandraTestContainer.